make/langtools/src/classes/build/tools/symbolgenerator/CreateSymbols.java

```diff
@@ -301,6 +302,45 @@ public void createSymbols(String ctDescriptionFileExtra, String ctDescriptionFil
         }
     }
 
+    private static final String PREVIEW_FEATURE_ANNOTATION_OLD =
+            "Ljdk/internal/PreviewFeature;";
+    private static final String PREVIEW_FEATURE_ANNOTATION_NEW =
+            "Ljdk/internal/javac/PreviewFeature;";
+    private static final String PREVIEW_FEATURE_ANNOTATION_INTERNAL =
+            "Ljdk/internal/PreviewFeature+Annotation;";
+    public static final Set<String> HARDCODED_ANNOTATIONS = new HashSet<>(
```

Comment:
shouldn't PREVIEW_FEATURE_ANNOTATION_INTERNAL be added to HARDCODED_ANNOTATIONS too?

Reply:
The *_INTERNAL annotation(s) only exists in ct.sym and are injected after the annotations are filtered, so I think it does not need to be in `HARDCODED_ANNOTATION`.